README.md

```diff
@@ -10,21 +10,49 @@ This project also implements ERC-3156 wrappers for the flash loan functionality
 
```

Comment:
When executing a flash loan you will need to code and deploy your own contract to receive the `IERC3156Borrower(receiver).onFlashLoan(parameters)` callback. You can call the `flashLoan(parameters)` function on any `ERC3156` compliant contract, such as the wrappers mentioned below, specifying your callback-receiving contract as the `receiver`.